chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,199 @@
+# Pre-trained models for text-to-speech
+
+Text-to-speech task (also called _speech synthesis_) comes with a range of challenges.
+
+First, just like in the previously discussed automatic speech recognition, the alignment between text and speech can be tricky.  
+The model has to learn to generate the correct duration and timing for each phoneme, word, or sentence which can be challenging, 
+especially for long and complex sentences.
+
+Next, there's the long-distance dependency problem: language has a temporal aspect, and understanding the meaning of a 
+sentence often requires considering the context of surrounding words. Ensuring that the TTS model captures and retains 
+contextual information over long sequences is crucial for generating coherent and natural-sounding speech.
+
+Finally, training TTS models typically requires pairs of text and corresponding speech recordings. On top of that, to ensure 
```

Comment:
Wonder if this can go in the section on fine-tuning? Feels more relevant there

Reply:
It makes sense to motivate here so that whilst people play with a different assortment of models, they have it in mind that, we can't just throw `any` paired dataset over.